app/grandchallenge/workstation_configs/migrations/0008_auto_20210920_1439.py

```diff
@@ -0,0 +1,47 @@
+# Generated by Django 3.1.13 on 2021-09-20 14:39
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        (
+            "workstation_configs",
+            "0007_workstationconfig_auto_jump_center_of_gravity",
+        )
+    ]
+
+    operations = [
+        migrations.RemoveField(
```

Comment:
We can no longer remove fields straight up, instead, we need to go through a process of deprecation (https://www.caktusgroup.com/blog/2021/05/25/django-migrations-and-deployment/). However, these views are rarely called, so a little downtime here is acceptable in this instance.